data/mods/Tanks/snippets.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Tank drone, meet the real deal.  See how you handle 120 millimeters of HELL YEAH!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"big fucking gun, earplugs are good for your brain\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"I have a bicyle-mounted tank cannon.  Your argument is invalid.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Next person to call this infantry fighting vehicle a \"tank\" is walking home.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Found what used to be an armored platoon.  Most tanks have hatches up top, not in the back like the new ones they're using.\"" ]
+  }
```

Comment:
As I suspected, yes. Does seem to be the least messy layout given current restrictions.